src/components/ProductDetail/ProductDetailContent.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+
+import "./ProductDetailContent.css";
+import errorImage from "../../assets/images/img_default.png";
+import kebabIcon from "../../assets/images/ic_kebab.png";
+import likeIconEmpty from "../../assets/images/ic_heart_empty.png";
+import likeIconFull from "../../assets/images/ic_heart_full.png";
+
+import Tag from "./Tag";
+
+const onErrorImg = (e) => {
+  e.target.src = errorImage;
+};
+
+function ProductDetailContent({ product }) {
+  const {
+    id,
+    name,
+    description,
+    price,
+    images,
+    favoriteCount,
+    tags = [],
+  } = product;
+
+  const [isButtonClicked, setIsButtonClicked] = useState(false);
+  const handleButtonClick = () => {
+    setIsButtonClicked(!isButtonClicked);
+  };
+
+  return (
+    <div className="contents-wrapper">
+      <img
+        className="image"
+        src={images}
+        alt={name}
+        width="486px"
+        height="486px"
+        onError={onErrorImg}
+      ></img>
```

Comment:
p5;
셀프 클로징 태그를 사용해도 괜찮을 것 같아요!
```suggestion
      />
```